uosc.lua

```diff
@@ -3169,9 +3198,9 @@ function open_file_navigation_menu(_directory, handle_select, menu_options)
 	table.sort(directories, word_order_comparator)
 
 	-- Pre-populate items with parent directory selector if not at root
-	local items = is_root and {} or {
-		{title = '..', hint = 'parent dir', value = directory.dirname}
-	}
+	items = is_root
+		and (function() return state.os == 'windows' and {{title = '..', hint = 'Drives', value = '$drives'}} or {} end)()
```

Comment:
Why is this a function and not just a parentheses?

Reply:
Are you saying `(state.os == 'windows' and {{title = '..', hint = 'Drives', value = '$drives'}} or {})`?

I tried it, and it does work. Since I'm very new to Lua, I didn't think of some tricks.

I will change it.